Remotion/ObjectBinding/Web/UI/Controls/BocTextValueImplementation/BocTextValueAutocompleteAttribute.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Remotion.ObjectBinding.Web.UI.Controls.BocTextValueImplementation
+{
+  /// <summary>
+  /// Defines possible standard autocomplete attributes to be used with <see cref="TextBoxStyle.AutoComplete"/>.
+  /// For a clear documentation about all the attributes, please visit the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete">MDN docs</a> for the HTML attribute.
+  /// </summary>
+  public class BocTextValueAutocompleteAttribute
```

Comment:
We should convert all of those variable names to `PascalCase` to fit the C# naming conventions. So `one_time_code` -> `OneTimeCode`.

Notepad++ has case conversions built in so you just need to copy out the names and paste them back in again.

Reply:
Don't forget the property name itself, too. Autocomplete -> AutoComplete